### PR TITLE
Print the QR code in the terminal

### PR DESCRIPTION
**Pull request by wyhaya**
Complete #45

---

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -355,3 +383,45 @@ fn error_404(req: &HttpRequest<crate::MiniserveConfig>) -> Result<HttpResponse,
         .into_string(),
     ))
 }
+
+// Print the QR code of the specified content in the terminal
+// For the width and height to be equal, two characters are used in each grid
+// Cannot control the line-height of the terminal, so it looks imperfect
+fn print_qrcode(content: String) {
```
Comment (svenstaro) [inline src/main.rs:390]: How about using something like this instead? https://github.com/svenstaro/qrcode-rust/blob/master/examples/encode_unicode.rs

Much less code for us and super clean unicode output.

File: src/main.rs [hunk 2]
```diff
@@ -355,3 +383,45 @@ fn error_404(req: &HttpRequest<crate::MiniserveConfig>) -> Result<HttpResponse,
...
+            }
+        }
+        text.push_str("\n");
+    }
+
+    print!("{}", text);
+}
+
+fn get_intranet_ip() -> Option<IpAddr> {
+    pnet_datalink::interfaces()
```
Comment (svenstaro) [inline src/main.rs:420]: Why exactly did you pull in this dependency?

File: src/main.rs (hunk 1 above)
Comment (wyhaya) [reply src/main.rs:390]: Looks good, should we use `qrcode` create?

File: src/main.rs (hunk 2 above)
Comment (wyhaya) [reply src/main.rs:420]: Get local IP address.

File: src/main.rs (hunk 1 above)
Comment (svenstaro) [reply src/main.rs:390]: Yeah, then we should also change the current qrcodegen dependency to that.

File: src/main.rs (hunk 2 above)
Comment (svenstaro) [reply src/main.rs:420]: I'm concerned about this apparently requring winpcap on Windows which think we can't link statically. I'd very much like this to stay a single static binary on Windows. I just checked this with `cargo-cross` but you could also check this natively.

Comment (startewho) [reply src/main.rs:420]: Myabe the crate [systemstat](https://github.com/myfreeweb/systemstat) will be an option. it support get the interfaces on diffferent os (contion windwos also).

**Pull request closed without merging**